veracruz-server/src/cli.rs

```diff
@@ -47,27 +46,15 @@ fn main() {
     };
     info!("Loaded policy {}", policy.policy_hash().unwrap_or("???"));
 
-    // create Actix runtime
-    let sys = actix_rt::System::new();
-
     // create Veracruz Server instance
-    let veracruz_server = match veracruz_server::server::server(&policy_json) {
-        Ok(veracruz_server) => veracruz_server,
-        Err(err) => {
-            eprintln!("{}", err);
-            process::exit(1);
-        }
-    };
+    veracruz_server::server::server(&policy_json).unwrap();
```

Comment:
Have you run anything using the `server-cli` with this PR? I haven't run it in a while, so no judgement if you haven't. Just curious.

Reply:
This gets checked by `quickstart` test in CI.